src/components/popper/__tests__/PopperComponent.test.js

```diff
@@ -50,4 +50,18 @@ describe('components/popper/PopperComponent', () => {
         expect(popperWrapper.length).toBe(1);
         expect(popperWrapper.prop('placement')).toBe('bottom-end');
     });
+
+    test('should apply the positionFixed to the Popper', () => {
```

Comment:
this test is testing react itself. you create the wrapper on line 56, then test what is on line 56. we want to test the associated logic, so it would be good for a test to cover when `isOpen` is false but `positionFixed` is true.

Reply:
When `isOpen` is false, we should not render the inner `<Popper>` component. Test is already implemented here: https://github.com/box/box-ui-elements/blob/master/src/components/popper/__tests__/PopperComponent.test.js#L16

So here I'm trying to test that when we pass in `positionFixed` to `<PopperComponent>` that we maintain that prop in the inner `<Popper>` (which is the dropdown menu).